sys-kernel/vanilla-kernel/vanilla-kernel-5.4.1.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit mount-boot savedconfig toolchain-funcs
+
+MY_P=linux-${PV}
+DESCRIPTION="Linux kernel built from vanilla upstream sources"
+HOMEPAGE="https://www.kernel.org/"
+SRC_URI="https://cdn.kernel.org/pub/linux/kernel/v5.x/${MY_P}.tar.xz"
+S=${WORKDIR}/${MY_P}
+
+LICENSE="GPL-2"
+SLOT="${PV}"
+KEYWORDS="~amd64"
+IUSE="+initramfs"
+
+# install-DEPEND actually
+RDEPEND="sys-apps/debianutils
+	initramfs? ( sys-kernel/dracut )"
+# we steal default configs from genkernel
+BDEPEND="sys-kernel/genkernel"
```

Comment:
do we really need to hard-depend on genkernel in order to steal their config? We can host it somewhere else (like devspace)

Reply:
Resolved via fetching Arch's config.